tools/make-self-signed-cert.sh

```diff
@@ -2,7 +2,8 @@
 
 set -x
 
-mkdir -p ssl
-openssl genrsa -out ssl/privatekey.pem 2048
-openssl req -new -sha256 -key ssl/privatekey.pem -out ssl/csr.pem -subj '/CN=www.toizom.com/O=MozillaIoT Gateway/C=US'
-openssl x509 -req -in ssl/csr.pem -signkey ssl/privatekey.pem -out ssl/certificate.pem
+export SSL_DIR="${HOME}/mozilla-iot/data/ssl"
```

Comment:
Does this need to be exported?

Reply:
And there really needs to be a way to override this. My mozilla-iot directory isn't in my HOME directory, it's in my Dropbox folder.